### PR TITLE
Changes for MTA version 5.0.1

### PR DESCRIPTION
**Pull request by RichardHoch**
Changes:
1.  Description of ulimit items Known Issues changed to indicate they were resolved in version 5.0.1 (as per Phil's request).
2. {ProductVersion} changed to 5.0.1 -- only impact on document is to change the name of file downloaded for installation of the Eclipse plugin. Changed confirmed by Phil.
3. Resolved https://issues.redhat.com/browse/MIGENG-829 by removing the limits to ulimit that were in the documentation.

### CODE REVIEW

**Comment by RichardHoch** [comment]
@apinnick : Please review.

File: docs/topics_5/templates/document-attributes.adoc [hunk 1]
```diff
@@ -35,7 +35,7 @@
 // * Doc metadata *
 // ****************
 :DocInfoProductName: Migration Toolkit for Applications
-:DocInfoProductNumber: 5.0
+:DocInfoProductNumber: 5.0.1
 :DocInfoProductNameURL: migration_toolkit_for_applications
```
Comment (apinnick) [inline docs/topics_5/templates/document-attributes.adoc:39]: Not sure this is a good change. Changing from 2 to 3 digits is problematic. If you were changing `5.0.0` to `5.0.1`, I would have no problem with that. You need to check how this attribute is used in the docs.

Comment (RichardHoch) [reply docs/topics_5/templates/document-attributes.adoc:39]: The only place it appears in text users see is in the master.adoc of release-notes-5.0. it also appears in the master-docinfo.xml files. If we do not make this change, then the opening sentence of the release notes will refer to version 5.0, not 5.0.1. If that's OK, I'll remove the change.

File: docs/topics_5/templates/document-attributes.adoc [hunk 2]
```diff
@@ -35,11 +35,11 @@
...
 
 :OpenShiftProductNumber: 4.5
-:ProductVersion: 5.0.0
+:ProductVersion: 5.0.1
```
Comment (RichardHoch) [inline docs/topics_5/templates/document-attributes.adoc:42]: The only impact this change will have in the user guide is in the name of the file to download to install for the IDE/Eclipse plug-in. Phil confirmed that this change was OK.

File: docs/topics_5/web-install-zip.adoc [hunk 3]
```diff
@@ -16,10 +16,10 @@ macOS has the following additional prerequisites:
 
 * Set the `maxproc` value to `2048` or greater.
 
-ifeval::["{ProductVersion}" == "5.0.0"]
+ifeval::["{ProductVersion}" == "5.0.1"]
```
Comment (RichardHoch) [inline docs/topics_5/web-install-zip.adoc:19]: @apinnick Does this change require QE confirmation? I think not but I want tie sure.

File: docs/topics_5/templates/document-attributes.adoc (hunk 1 above)
Comment (apinnick) [reply docs/topics_5/templates/document-attributes.adoc:39]: Release notes are for y versions, not individual z versions. So remove this change.

File: docs/topics_5/templates/document-attributes.adoc (hunk 2 above)
Comment (apinnick) [reply docs/topics_5/templates/document-attributes.adoc:42]: Have you confirmed that the maxfiles value change is OK for 5.0.1?

File: docs/topics_5/web-install-zip.adoc (hunk 3 above)
Comment (apinnick) [reply docs/topics_5/web-install-zip.adoc:19]: Yes, everything should be checked with QE.

Comment (apinnick) [reply docs/topics_5/web-install-zip.adoc:19]: Why would 5.0.1 not be checked with QE?

File: docs/topics_5/templates/document-attributes.adoc (hunk 1 above)
Comment (RichardHoch) [reply docs/topics_5/templates/document-attributes.adoc:39]: Removed.

File: docs/topics_5/web-install-zip.adoc (hunk 3 above)
Comment (RichardHoch) [reply docs/topics_5/web-install-zip.adoc:19]: @PhilipCattanach , @mrizzi 

Please check if "5.0.1" is correct here.

**Comment by PhilipCattanach** [comment]
PR now linked to WINDUP-2751

File: docs/topics_5/web-install-zip.adoc [hunk 4]
```diff
@@ -12,17 +12,10 @@ The {WebName} has been tested with Chrome and Firefox.
 * OpenJDK 1.8, OpenJDK 11, Oracle JDK 1.8, or Oracle JDK 11
 * 16 GB RAM
 
-macOS has the following additional prerequisites:
+macOS has the following additional prerequisite:
 
 * Set the `maxproc` value to `2048` or greater.
 
-ifeval::["{ProductVersion}" == "5.0.0"]
```
Comment (apinnick) [inline docs/topics_5/web-install-zip.adoc:19]: There are 3 other modules that need to be updated because they also have maxfiles

Comment (RichardHoch) [reply docs/topics_5/web-install-zip.adoc:19]: Done.

Comment (apinnick) [reply docs/topics_5/web-install-zip.adoc:19]: BTW, it's insane to update 4 different files. See if you can modularize this

**Pull request closed without merging**